jekyll/_cci2/server/v4.2/operator/managing-orbs.adoc

```diff
@@ -0,0 +1,78 @@
+---
+contentTags:
+  platform:
+    - Server v4.2
+    - Server Admin
+---
+= Managing Orbs
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Use this page to learn about orbs and how to manage them within CircleCI server v4.x.
+:icons: font
+:toc: macro
+:toc-title:
+
+This section describes how to manage orbs for an installation of server v4.x. Server installations include their own local orb registry. All orbs referenced in configs refer to the orbs in the server orb registry. You are responsible for maintaining orbs. This includes copying orbs from the public registry, updating orbs that may have been previously copied, and registering your company's private orbs, if they exist.
```

Comment:
Just checking if "server v4.x" needs to be changed to the specific version or if this typically uses a variable